docs/concepts/OpenFunction-events-framework.md

```diff
@@ -72,19 +72,45 @@ In this sample, the event source is a Kafka server and the target function is a
 
 - A Knative runtime function (target function)
 
-  You can refer to [Sample Function Go](https://github.com/OpenFunction/samples/tree/main/functions/Knative/hello-world-go) to create a Knative runtime function.
+  You can apply the following configuration to create a function that will be used to be triggered.
 
-  Here I assume that the name of this function (Knative Service) is `function-sample-serving-qrdx8-ksvc-fwml8` . You should be able to see this value with the `kubectl get ksvc` command
+  ```yaml
+  apiVersion: core.openfunction.io/v1beta1
+  kind: Function
+  metadata:
+    name: sink-a
+  spec:
+    version: "v1.0.0"
+    image: "openfunction/sink-sample:latest"
+    port: 8080
+    serving:
+      runtime: "knative"
+      template:
+        containers:
+          - name: function
+            imagePullPolicy: Always
+  ```
+
+  You can get the URL address of the function with the following command:
+
+  > Please note that when you use this URL, you need to extend it to the following style:
```

Comment:
`openfunction.io` is ok for most of the cases and so does `openfunction.io.svc`: 

https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/ 
`In summary, a pod in the test namespace can successfully resolve either data.prod or data.prod.svc.cluster.local`

User might customize cluster.local to another customized domain suffix on the contrary.

Just mention `openfunction` is a service in namespace `io` is all right.

Reply:
ok